### ISSUE TITLE
the difference between train_set and val_set

### ISSUE THREAD

**Issue by aha831** [kwea123/nsff_pl#17]
Your training set and your test set seem to overlap. Are the published metrics credible？

<img width="668" alt="image" src="https://user-images.githubusercontent.com/60975608/226588906-e31fadf5-791c-457d-9cac-a1742273c0de.png">
